tests/data/no_triggers_project/parameters.yml

```diff
@@ -0,0 +1,208 @@
+        lane_capacity: 900
+        lanes: 2
+        link_type_id: y
+  modes:
+  - car:
+      description: All motorized vehicles
+      letter: c
+  - transit:
+      description: All links capable of supporting transit-vehicles
+      letter: t
+  - walk:
+      description: Linkes where walking is permitted
+      letter: w
+  - bicycle:
+      description: All Bicycles
+      letter: b
+  nodes:
+    fields:
+    - node_id:
+        description: node_id. THIS FIELD CANNOT BE CHANGED
+        type: integer
+    - is_centroid:
+        description: is_centroid
+        type: integer
+  osm:
+    all_link_types:
+    - secondary_link
+    - escalator
+    - trail
+    - cycleway
+    - path
+    - trunk_link
+    - secondary
+    - escape
+    - track
+    - road
+    - motorway_link
+    - primary
+    - corridor
+    - residential
+    - footway
+    - motorway
+    - primary_link
+    - unclassified
+    - bus_guideway
+    - tertiary_link
+    - living_street
+    - pedestrian
+    - bridleway
+    - elevator
+    - motor
+    - trunk
+    - tertiary
+    - service
+    - steps
+    - proposed
+    - raceway
+    - construction
+    - abandoned
+    - platform
+    - unclassified
+    modes:
+      bicycle:
+        link_types:
+        - cycleway
+        - corridor
+        - pedestrian
+        - path
+        - track
+        - trail
+        - unclassified
+        mode_filter:
+          bicycle: 'no'
+        unknown_tags: true
+      car:
+        link_types:
+        - motor
+        - motorway
+        - trunk
+        - primary
+        - secondary
+        - tertiary
+        - unclassified
+        - residential
+        - motorway_link
+        - trunk_link
+        - primary_link
+        - secondary_link
+        - tertiary_link
+        - living_street
+        - service
+        - escape
+        - road
+        mode_filter:
+          motor_vehicle: 'no'
+        unknown_tags: true
+      transit:
+        link_types:
+        - motor
+        - motorway
+        - trunk
+        - primary
+        - secondary
+        - tertiary
+        - unclassified
+        - residential
+        - motorway_link
+        - trunk_link
+        - primary_link
+        - secondary_link
+        - tertiary_link
+        - living_street
+        - service
+        - escape
+        - road
+        - bus_guideway
+        unknown_tags: true
+      walk:
+        link_types:
+        - cycleway
+        - footway
+        - steps
+        - corridor
+        - pedestrian
+        - elevator
+        - escalator
+        - path
+        - track
+        - trail
+        - bridleway
+        - unclassified
+        mode_filter:
+          pedestrian: 'no'
+        unknown_tags: true
+system:
+  cpus: 0
+  default_directory: C:\Users\pedro.camargo\Documents\sourcecode\drt
+  driving side: right
+  logging: true
+  logging_directory: D:/no_triggers_project
+  spatialite_path: C:\Users\pedro\Documents\mod_spatialite-NG-win-amd64
```

Comment:
Do these hardcoded paths cause problems?

Reply:
They don't. There are workflows for them to be changed, and one of them will be deprecated with the upcoming version